### PR TITLE
Add high level interface of the Zivid SDK for the WRS system

### PR DESCRIPTION
**Pull request by chenhaox**
1. Add a high-level interface for the Zivid SDK.
2. Add a simple example to use the Zivid interface

### CODE REVIEW

File: drivers/devices/zivid/zivid_sdk.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
+"""
+A high-level interface of the Zivid SDK for the WRS system
+
```
Comment (wanweiwei07) [inline drivers/devices/zivid/zivid_sdk.py:3]: 多余空行删除

File: drivers/devices/zivid/zivid_sdk.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+Prerequisite:
+    Install Zivid  SDK:
+        "pip install zivid"
+
+Reference: https://support.zivid.com/en/latest/index.html
+"""
+from pathlib import Path
+import datetime
+
```
Comment (wanweiwei07) [inline drivers/devices/zivid/zivid_sdk.py:16]: 多余空行

File: drivers/devices/zivid/zivid_sdk.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        filters.experimental.contrast_distortion.removal.enabled = False
+        filters.experimental.contrast_distortion.removal.threshold = 0.5
+        color = settings.processing.color
+        color.balance.red = 1.0
+        color.balance.blue = 1.0
+        color.balance.green = 1.0
+        color.gamma = 1.0
+        settings.processing.color.experimental.mode = "automatic"
+        print(settings.processing)
+
```
Comment (wanweiwei07) [inline drivers/devices/zivid/zivid_sdk.py:42]: 在同一个函数里尽量不用空行

File: drivers/devices/zivid/zivid_sdk.py [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    def __del__(self):
+        self.release()
+
+
+if __name__ == "__main__":
+    import cv2
+
+    cam = Zivid()
+    pcd, img = cam.get_pcd_rgba()
```
Comment (wanweiwei07) [inline drivers/devices/zivid/zivid_sdk.py:169]: 返回值是三个参数？

File: drivers/devices/zivid/zivid_sdk.py (hunk 1 above)
Comment (chenhaox) [reply drivers/devices/zivid/zivid_sdk.py]: Resolved

File: drivers/devices/zivid/zivid_sdk.py (hunk 3 above)
Comment (chenhaox) [reply drivers/devices/zivid/zivid_sdk.py]: Solved

**Pull request merged**